migration-from-4x.md

```diff
@@ -79,7 +79,9 @@ tinymce.init({
     editor.ui.registry.addMenuButton('insert', {
       icon: 'plus',
       tooltip: 'Insert',
-      fetch: (callback) => callback('image link | inserttable')
+      fetch: function (callback) {
+        callback('image link | inserttable')
```

Comment:
Missing semicolon at the end of the line here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d